### ISSUE TITLE
Spring Batch integration fails for multiline records bound to a collection or map

### ISSUE THREAD

**Issue by GoogleCodeExporter** [unlockha/beanio#89]
```
See forum post: https://groups.google.com/forum/#!topic/beanio/p3DEIO0NoQg
```

Original issue reported on code.google.com by `kevin.s...@gmail.com` on 12 Jul 2013 at 2:24

**Comment by GoogleCodeExporter** [comment]
```
Fixed for 2.0.7.
```

Original comment by `kevin.s...@gmail.com` on 14 Jul 2013 at 5:25

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.s...@gmail.com` on 7 Sep 2013 at 6:19
- Changed state: **Fixed**
